CSharpSourceCode/Abilities/Scripts/AbilityScript.cs

```diff
@@ -30,7 +31,9 @@ internal void SetAgent(Agent agent)
         protected override bool MovesEntity() => true;
         protected virtual bool ShouldMove()
         {
-            return _ability.Template.AbilityEffectType != AbilityEffectType.TargetedStaticAOE && _ability.Template.AbilityEffectType != AbilityEffectType.CenteredStaticAOE;
+            return _ability.Template.AbilityEffectType != AbilityEffectType.TargetedStaticAOE &&
```

Comment:
not for now, but potentially we should work with flags... it reads really bad

Reply:
yeah, i agree